main.go

```diff
@@ -64,9 +64,8 @@ func main() {
 	})
 
 	http.HandleFunc("/events", func(w http.ResponseWriter, req *http.Request) {
-		resp, err := auditlog.FindAll(auditlog.EventsRequestParams{
-			End:    time.Now(),
-			Action: "user.login",
+		resp, err := events.FindAll(auditlog.ListRequestParams{
```

Comment:
`Iter` and `ListRequestParams` are both reusable structs that can be used to paginate over varying data. By not including `ListRequestParams` to the `events` package we can reuse it if there is another entity that requires pagination.

Reply:
It feels a little soon for an abstraction, might I suggest having the `events` package expose `ListRequestParams`? Once there are more resources to iterate through it would make more sense to DRY it up.